### PR TITLE
Let op /ctf and /ctr to any group

### PR DESCRIPTION
**Pull request by Maxopoly**
Adressing https://github.com/Civcraft/Citadel/issues/116

@rourke750

### CODE REVIEW

**Comment by rourke750** [comment]
Could you also make it to check if the player has the citadel.admin permission.  Also thx.

**Comment by rourke750** [comment]
Lol also can you increment the pom version and then just merge all your commits into one.  Thank you.

**Comment by ttk2** [comment]
so this is ready?

**Comment by ttk2** [comment]
ok to test

**Comment by rourke750** [comment]
No it isnt

File: src/vg/civcraft/mc/citadel/command/commands/Fortification.java [hunk 1]
```diff
@@ -62,12 +62,12 @@ public boolean execute(CommandSender sender, String[] args) {
...
 		
 		PlayerType type = g.getPlayerType(uuid);
-		if (type == null){
+		if (!p.hasPermission("citadel.admin") && !p.isOp() && type == null){
 			p.sendMessage(ChatColor.RED + "You are not on this group.");
 			return true;
 		}
 		GroupPermission gPerm = gm.getPermissionforGroup(g);
-		if (!gPerm.isAccessible(type, PermissionType.BLOCKS)){
+		if (!p.hasPermission("citadel.admin") && !p.isOp() && !gPerm.isAccessible(type, PermissionType.BLOCKS)){
```
Comment (rourke750) [inline src/vg/civcraft/mc/citadel/command/commands/Fortification.java]: can you change this to !(p.hasPermission("citadel.admin") || p.isOp()) && !gPerm.isAccessible(type, PermissionType.BLOCKS)

File: src/vg/civcraft/mc/citadel/command/commands/Fortification.java [hunk 2]
```diff
@@ -62,12 +62,12 @@ public boolean execute(CommandSender sender, String[] args) {
 		}
 		
 		PlayerType type = g.getPlayerType(uuid);
-		if (type == null){
+		if (!p.hasPermission("citadel.admin") && !p.isOp() && type == null){
```
Comment (rourke750) [inline src/vg/civcraft/mc/citadel/command/commands/Fortification.java]: can you change this to !(p.hasPermission("citadel.admin") || p.isOp()) && type == null

File: src/vg/civcraft/mc/citadel/command/commands/Reinforce.java [hunk 3]
```diff
@@ -56,12 +56,12 @@ public boolean execute(CommandSender sender, String[] args) {
 			return true;
 		}
 		PlayerType type = g.getPlayerType(uuid);
-		if (type == null){
+		if (!p.hasPermission("citadel.admin") && !p.isOp() && type == null){
```
Comment (rourke750) [inline src/vg/civcraft/mc/citadel/command/commands/Reinforce.java]: same thing here.

File: src/vg/civcraft/mc/citadel/command/commands/Reinforce.java [hunk 4]
```diff
@@ -56,12 +56,12 @@ public boolean execute(CommandSender sender, String[] args) {
...
 			p.sendMessage(ChatColor.RED + "You are not on this group.");
 			return true;
 		}
 		GroupPermission gPerm = gm.getPermissionforGroup(g);
-		if (!gPerm.isAccessible(type, PermissionType.BLOCKS)){
+		if (!p.hasPermission("citadel.admin") && !p.isOp() && !gPerm.isAccessible(type, PermissionType.BLOCKS)){
```
Comment (rourke750) [inline src/vg/civcraft/mc/citadel/command/commands/Reinforce.java]: Same thing here.

File: src/vg/civcraft/mc/citadel/command/commands/Fortification.java (hunk 2 above)
Comment (ProgrammerDan) [inline src/vg/civcraft/mc/citadel/command/commands/Fortification.java]: D'Morgan's law for the win.

File: src/vg/civcraft/mc/citadel/command/commands/Fortification.java (hunk 1 above)
Comment (Maxopoly) [inline src/vg/civcraft/mc/citadel/command/commands/Fortification.java]: I'll change it if you want me to, but what's the advantage of that? It does all 3 checks in 99,9 % of the times this is run anyway.

Comment (rourke750) [inline src/vg/civcraft/mc/citadel/command/commands/Fortification.java]: Oh you're right my bad.

Comment (Maxopoly) [inline src/vg/civcraft/mc/citadel/command/commands/Fortification.java]: So merge?

**Pull request merged**